doc/sphinx-guides/source/developers/deployment.rst

```diff
@@ -0,0 +1,72 @@
+==========
+Deployment
+==========
+
+Developers often only deploy Dataverse to their :doc:`dev-environment` but it can be useful to deploy Dataverse to cloud services such as Amazon Web Services (AWS).
+
+.. contents:: |toctitle|
+	:local:
+
+Deploying Dataverse to Amazon Web Services (AWS)
+------------------------------------------------
+
+We have written scripts to deploy Dataverse to Amazon Web Services (AWS) but they require some setup.
+
+Install AWS CLI
+~~~~~~~~~~~~~~~
+
+First, you need to have AWS Command Line Interface (AWS CLI) installed, which is called ``aws`` in your terminal. Launching your terminal and running the following command to print out the version of AWS CLI will tell you if it is installed or not:
+
+``aws --version``
```

Comment:
It might be helpful to mention a known-working version of the AWS CLI (which may make things easier if a future update changes argument formatting used by the EC2 scripts).

Reply:
Mentioning a known good version isn't a bad idea but I believe the idea is that we'll be using this script fairly regularly so we should notice if it stops working.